jdi-light-html-tests/src/test/java/io/github/epam/example/complex/ComboboxExampleTests.java

```diff
@@ -0,0 +1,73 @@
+package io.github.epam.example.complex;
+
+import io.github.epam.TestsInit;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static io.github.com.StaticSite.html5Page;
+import static io.github.com.pages.HtmlElementsPage.iceCreamIs;
+import static io.github.epam.html.tests.elements.complex.enums.IceCreamFlavors.Strawberry;
+import static io.github.epam.html.tests.elements.complex.enums.IceCreamFlavors.Vanilla;
+import static io.github.epam.html.tests.site.steps.States.shouldBeLoggedIn;
+import static java.util.Arrays.asList;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.is;
+
+public class ComboboxExampleTests implements TestsInit {
+
+    @BeforeMethod
+    public void before() {
+        shouldBeLoggedIn();
+        html5Page.shouldBeOpened();
+        iceCreamIs.select(text);
+    }
+    String text = "Coconut";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=4790788)